pkg/worker/worker.go

```diff
@@ -112,7 +112,7 @@ func (w *Worker) runWorker(ctx context.Context) {
 			if len(kinds) == 0 {
 				w.logger.Warn("no job-handler registered, skipping dequeue")
 			} else {
-				w.logger.Info("looking for a job", zap.Strings("kinds", kinds))
+				w.logger.Debug("looking for a job", zap.Strings("kinds", kinds))
```

Comment:
@spy16 also changed this to Debug. Or is there any other way we wanna handle this ?

Reply:
no. debug is fine i think